README.md

```diff
@@ -76,7 +76,8 @@
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-To see my portfolio click [here](https://news-website-f9vh.onrender.com)
+for live demo of the website click [here](https://news-website-f9vh.onrender.com)
```

Comment:
It would be nice to work on this before merging your PR, any HR out there would take a look at your live demo link.

As mentioned by the previous CR, your live demo link still not working, kindly fix it.
<img width="921" alt="image" src="https://github.com/Esmatullah12/news-website/assets/31897434/4e9daa3a-0600-467c-8746-631ba34825ad">


And also, your MIT link is still broken.
<img width="1083" alt="image" src="https://github.com/Esmatullah12/news-website/assets/31897434/5df0c5b4-3421-42da-b9fa-a758821161f2">